packages/editor/src/components/block-list/block.js

```diff
@@ -776,7 +774,25 @@ const applyWithDispatch = withDispatch( ( dispatch, ownProps, { select } ) => {
 	};
 } );
 
+const getBlock = memize( ( name, attributes ) => {
+	return {
+		name,
+		attributes,
+	};
+} );
+
 export default compose(
+	// This withSelect is only necessary for backward compatibility
+	// Users of the editor.BlockListBlock filter used to be able to access the block prop
+	// Ideally these blocks would rely on the clientId prop only.
+	withSelect( ( select, { clientId } ) => {
```

Comment:
Prior to #11899, `withFilters( 'editor.BlockListBlock' )` occurred after `applyWithSelect` . Could we / shouldn't we do the same here, rather than creating a separate `withSelect` (which will also incur another per-block store subscription)?

Reply:
Restored the order.